Sources/Subs.php

```diff
@@ -829,6 +829,9 @@ function timeformat($log_time, $show_today = true, $offset_type = false, $proces
 	if ($context['server']['is_windows'] && strpos($str, '%e') !== false)
 		$str = str_replace('%e', ltrim(strftime('%d', $time), '0'), $str);
 
+	// fix unkown %l with %I
```

Comment:
well i forgot to mention my motivation for this "fix",
when i keep the %L(small) strftime return false instead of a string mit date and time:

```
<?php
echo 'test';
$test = strftime('%l:%M am');
var_dump($test);
```
->
```
test

D:\Programme\xampp2\htdocs\test.php:4:boolean false
```

when i use %i(high) 

```
<?php
echo 'test';
$test = strftime('%I:%M am');
var_dump($test);
```

```
test

D:\Programme\xampp2\htdocs\test.php:4:string '05:42 am' (length=8)
```


So on my side strftime doesn't work with %L ...